common/meta/raft-store/src/state_machine/sm_meta_api_impl.rs

```diff
@@ -118,12 +120,12 @@ impl MetaApi for StateMachine {
 
         let it = self.database_lookup().range(..)?;
         for r in it {
-            let (db_name, seq_id) = r?;
+            let (db_lookup_key, seq_id) = r?;
             let seq_meta = self.get_database_meta_by_id(&seq_id.data)?;
 
             let db_info = DatabaseInfo {
                 database_id: seq_id.data,
-                db: db_name,
+                db: db_lookup_key.get_database_name(),
                 meta: seq_meta.data,
             };
```

Comment:
Do we need `tenant` in `DatabaseInfo`? 🤔 
@BohuTANG

Reply:
No need now